src/webgl.js

```diff
@@ -953,9 +944,9 @@ function resizeCanvas(canvas) {
 }
 
 function fitBoundingBox(w, h, canvasW, canvasH) {
-  const left = Math.round((canvasW - w) / 2.0);
+  const left = Math.round((canvasW - w) * 0.5);
   const right = canvasW - left;
-  const top = Math.round((canvasH - h) / 2.0);
+  const top = Math.round((canvasH - h) * 0.5);
```

Comment:
Not a bad thing, but remember JS versus GLSL follow different rules: 
```
const fitBoundingBox = (w, h, cw, ch) => [
    left = Math.round((cw - w)/2),
    above = Math.round((ch - h)/2),
    cw - left,
    ch - above
]
```

(note: the above trick only works if your inline vars are not already assigned to system constants... `top` is browser-problematic, apparently)

Reply:
I don't think we can create the vars in the return, if I'm wrong on that though let me know. I've updated it to use the arrow function and compute the second two return values instead of assigning them to vars.